content/en/docs/attestation/attestation-service/quick-start/_index.md

```diff
@@ -0,0 +1,437 @@
+---
+title: Quick Start
+description: Quick Start to deploy the Attestation Service
+weight: 2
+categories:
+- docs
+tags:
+- docs
+- attestation-service
+---
+
+For the latest deployment document, please refer to the CoCoAS [github page](https://github.com/confidential-containers/trustee/tree/main/attestation-service#quick-start).
```

Comment:
You can add words to `styles/config/vocabularies/coco/accept.txt` to make the spellcheck not complain. It's case sensitive. Follow the format already there for things that can be uppercase or lowercase.